packages/cli/__tests__/daemon/__unit__/DaemonClient.unit.test.ts

```diff
@@ -50,9 +50,9 @@ describe("DaemonClient tests", () => {
 
         const daemonClient = new DaemonClient(client as any, server, "fake");
         const daemonResponse: IDaemonResponse = {
-            argv: ["feed", "dog"],
+            argv: ["feed", "🐶"],
             cwd: "fake",
-            env: {},
+            env: { FORCE_COLOR: "0" },
```

Comment:
Quick question, but is there a reason we are forcing color here, and not in the other tests?

Reply:
No particular reason, just wanted to set an environment variable in at least one test to make sure it shows up correctly in the snapshot.

The name/value of the env var doesn't matter here, and we could add it to all tests in this file if you want 🙂

Reply:
No, I was just curious.